src/executables/csql_launcher.c

```diff
@@ -429,12 +429,7 @@ main (int argc, char *argv[])
     }
   else if (!csql_arg.sa_mode && csql_arg.skip_vacuum)
     {
-      /* Don't allow to skip vacuum on CS mode
-         goto print_usage;
-         }
-         else if (explicit_single_line && csql_arg.single_line_execution == false)
-         {
-         /* Don't allow both at once. */
+      /* Don't allow both at once. */
```

Comment:
The comments and indentation are wrong, so please recover your code like this:

/* Don't allow to skip vacuum on CS mode */
         goto print_usage;
  }
  else if (explicit_single_line && csql_arg.single_line_execution == false)
  {
         /* Don't allow both at once. */

Reply:
@mhoh3963 This removed part is already in the comment. So, I think if we need to re-insert this part, then how about making another PR for it? Because this PR is for fixing the compile error.

Reply:
OK, i will open another PR for the change to keep fixes separated by type.

Reply:
@adrianbrab We going to open a new CBRD issue for handling it. Please revert this.

Reply:
@hornetmj 
OK, I think good your suggestion.

@adrianbrab 
This is bug occurred with wrong comment made by me. So, please, don't modify this code. I will create new issue and fix it.

Reply:
@adrianbrab 
I already fixed codes in csql_launcher.c, so please revert the modified code and send PR again,

Reply:
merged with develop